test/integration/targets/vmware_guest_facts/tasks/main.yml

```diff
@@ -72,6 +72,8 @@
       - "guest_facts_0001['instance']['hw_folder'] is defined"
       - "guest_facts_0001['instance']['guest_question'] is defined"
       - "guest_facts_0001['instance']['guest_consolidation_needed'] is defined"
+      - "guest_facts_0001['instance']['moid'] is defined"
```

Comment:
_vimid is already provided by pyvmomi
_vimref is already provided by pyvmomi

If you are okay having these fields duplicated in the output, then merge it, otherwise have folks use these three _vim fields for the id, ref, and type provided by pyvmomi:

https://github.com/vmware/pyvmomi/blob/master/pyVmomi/VmomiSupport.py#L352